tests/test_schemas.py

```diff
@@ -54,27 +68,42 @@ def load_as_dataset(filepath):
     return iati.Dataset(xml_str)
 
 
-@pytest.mark.parametrize('filepath', list_xml_files_recursively('tests/activity-tests/should-pass/') +  # Legacy tests
-                         list_xml_files_recursively('tests/should-pass/'))  # Tests in new format
-def test_pass_files(activity_schema, filepath):
-    """Check that all 'should-pass' test files are XML and pass Schema validation."""
+@pytest.mark.parametrize('filepath', list_xml_files_recursively('tests/activity-tests/should-pass/') +  # Legacy activity test cases
+                         list_xml_files_recursively('tests/should-pass/iati-activities/') +  # Legacy activity test cases
+                         list_xml_files_recursively('tests/organisation-tests/should-pass/') +  # Legacy organisaion test cases
+                         list_xml_files_recursively('tests/should-pass/iati-organisations/'))  # Organisation test cases in new format
+def test_pass_files(activity_schema, organisation_schema, filepath):
+    """Check that all activity and organisation schema 'should-pass' test files are XML and pass Schema validation."""
     dataset = load_as_dataset(filepath)
 
+    if 'activity-tests' in filepath or 'iati-activities' in filepath:
```

Comment:
Same as previous comment about extracting this functionality and implicit `else`

Reply:
Resolved by merging `2.03dev` branch in f0aca6e.